pytest with no args does not honour the SOURCE_PATH

# Expected Behavior

I can use the pytest task with no args and a `SOURCE_PATH`

# Actual Behavior

When no args are passed, the command executed is

```bash
pytest "" <SOURCE_PATH> 
```

Which does not work as expected, as it ignores the path specified.

# Steps to Reproduce the Problem

Run the task with no args and a source path

# Additional Info

Running `tektoncd/website` unit tests.
The way the task does:

```bash
$ pytest "" sync
========================================================================= test session starts =========================================================================
platform darwin -- Python 3.8.12, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /System/Volumes/Data/go/src/github.com/tektoncd/website
collected 29 items / 3 errors / 26 selected

=============================================================================== ERRORS ================================================================================
______________________________________________ ERROR collecting tutorials/katacoda/getting-started/src/app/main_test.py _______________________________________________
ImportError while importing test module '/System/Volumes/Data/go/src/github.com/tektoncd/website/tutorials/katacoda/getting-started/src/app/main_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/Cellar/python@3.8/3.8.12/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tutorials/katacoda/getting-started/src/app/main_test.py:1: in <module>
    import main
tutorials/katacoda/getting-started/src/app/main.py:1: in <module>
    from flask import Flask, render_template, request
E   ModuleNotFoundError: No module named 'flask'
____________________________________________________ ERROR collecting tutorials/katacoda/runs/src/app/main_test.py ____________________________________________________
ImportError while importing test module '/System/Volumes/Data/go/src/github.com/tektoncd/website/tutorials/katacoda/runs/src/app/main_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/Cellar/python@3.8/3.8.12/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tutorials/katacoda/runs/src/app/main_test.py:1: in <module>
    import main
tutorials/katacoda/runs/src/app/main.py:1: in <module>
    from flask import Flask, render_template, request
E   ModuleNotFoundError: No module named 'flask'
__________________________________________________ ERROR collecting tutorials/katacoda/tasks/src/client/main_test.py __________________________________________________
tutorials/katacoda/tasks/src/client/main_test.py:4: in <module>
    with open('/etc/credential/apiKey') as f:
E   FileNotFoundError: [Errno 2] No such file or directory: '/etc/credential/apiKey'
======================================================================= short test summary info =======================================================================
ERROR tutorials/katacoda/getting-started/src/app/main_test.py
ERROR tutorials/katacoda/runs/src/app/main_test.py
ERROR tutorials/katacoda/tasks/src/client/main_test.py - FileNotFoundError: [Errno 2] No such file or directory: '/etc/credential/apiKey'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================================== 3 errors in 3.61s ==========================================================================
```

The correct way:

```bash
$ pytest sync
========================================================================= test session starts =========================================================================
platform darwin -- Python 3.8.12, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /System/Volumes/Data/go/src/github.com/tektoncd/website
collected 29 items

sync/test_sync.py .......................                                                                                                                       [ 79%]
sync/test_versions.py ......                                                                                                                                    [100%]

========================================================================= 29 passed in 9.15s ==========================================================================
```

--- a/task/pytest/0.1/pytest.yaml
+++ b/task/pytest/0.1/pytest.yaml
@@ -53,4 +53,8 @@ spec:
           fi
           pip install pytest
         fi
-        pytest "$(inputs.params.ARGS)" "$(inputs.params.SOURCE_PATH)"
+        if [ -z "$(inputs.params.ARGS)" ]; then
+          pytest "$(inputs.params.SOURCE_PATH)"
+        else
+          pytest "$(inputs.params.ARGS)" "$(inputs.params.SOURCE_PATH)"
+        fi

